cluster_setup/kubernetes/kubernetes_ansible.adoc

```diff
@@ -0,0 +1,308 @@
+[[kubernetes-ansible]]
+= {product-title} Kubernetes Ansible Cluster Setup
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+
+The link:https://github.com/kubernetes/contrib/tree/master/ansible[kubernetes/contrib] repo contains an Ansible playbook and set of roles that can be used to deploy a Kubernetes cluster using {product-title}. The hosts can be running on real hardware, VMs, things in a public cloud, etc. Anything that you can connect to via SSH. You can also use Vagrant to deploy the hosts on which to run this playbook. For more information, see the Vagrant Deployer section below.
+
+== Before starting
+
+* Record the IP address/hostname of which machine you want to be your master (only support a single master)
+* Record the IP address/hostname of the machine you want to be your etcd server (often same as master, can be more than one)
+* Record the IP addresses/hostname of the machines you want to be your nodes. (the master can also be a node)
+* Make sure your ansible running machine has ansible 1.9 and python-netaddr installed.
+
+== Setup
+
+=== Configure inventory
+
+Add the system information gathered above into a file called `inventory`,
+or create a new one for the cluster.
+Place the `inventory` file into the `./inventory` directory.
+
+For example:
+
+```sh
```

Comment:
Not really sh. Should we use ini? Also, this is Markdown style code fencing, which does work. Do we want to be strict about only using Asciidoc-style code fencing?